portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase

```diff
@@ -35,21 +35,75 @@ definition {
 	}
 
 	@description = "Automation ID: LRAC-8119 | Test Summary: Create Access Request"
-	@ignore = "true"
 	@priority = "5"
 	test CanCreateAccessRequest {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToDataControlAndPrivacy();
+
+		Click(locator1 = "ACDataControlAndPrivacy#RESQUEST_LOG");
+
+		Click(locator1 = "ACDataControlAndPrivacy#CREATE_REQUEST");
+
+		Click(
+			key_jobType = "Access",
+			locator1 = "ACDataControlAndPrivacy#JOB_TYPE");
+
+		Click(
+			key_fieldType = "Find by Email",
+			locator1 = "ACDataControlAndPrivacy#DATA_ID");
+
+		Click(locator1 = "ACDataControlAndPrivacy#FIND_BY_EMAIL_FIELD");
+
+		Click(
+			key_emailAdress = "Abram.Bauch@hotmail.com",
+			locator1 = "ACDataControlAndPrivacy#FIND_EMAIL_FIELD");
```

Comment:
I think I could make a `macro` for that. What do you think?

Reply:
fixed